kie-server-parent/kie-server-controller/kie-server-controller-impl/pom.xml

```diff
@@ -68,6 +68,11 @@
       <groupId>org.kie.server</groupId>
       <artifactId>kie-server-common</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kie.server</groupId>
+      <artifactId>kie-server-controller-rest</artifactId>
```

Comment:
This brings a cyclic dependency between impl and rest modules.

Reply:
1 if move the unit test to kie-server-controller-impl, need the dependncy.
2 the dependency scope is test.
So back to old module?

Reply:
I move the unit test to kie-server-controller-rest.